firmware/README.md

```diff
@@ -180,6 +181,14 @@ It is compared to the filtered inPhase values detected by the INS radar. Anythin
 
 The default level is set to 60. This is based on the radar testing documented [here](https://docs.google.com/document/d/12TLw6XE9CSaNpguytS2NCSCP0aZWUs-OncmaDdoTKfo/edit?usp=sharing).
 
+### STATE0_WINDOW_TIME
+
+This is defined via a macro in the stateMachine.h header file
+
+It is the length of time (or window) after the door_status changes from an open to a closed state where entrance to state1 is allowed. After the state 0 timer has surpassed this window, the state machine with still in state 0 until this timer reset (door open and close) 
```

Comment:
"With still in" should this be "will stay in"?

Reply:
yes, thanks for the catch